templates/search/results-contextual-en.html

```diff
@@ -1,20 +1,35 @@
 ---
 {
-	"title": "Search results (Contextual)",
+	"title": "Search (Contextual)",
 	"language": "en",
 	"altLangPage": "results-contextual-fr.html",
 	"pageType": "search",
 	"pageclass": "page-type-search",
-	"dateModified": "2022-08-02",
+	"dateModified": "2025-07-28",
 	"share": "false",
 	"deptfeature": false
 }
 ---
 
 <div class="well">
 	<form class="mrgn-tp-sm">
-		<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
-		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query aria-describedby="gc-pi">
+		<div class="row">
+			<div class="col-md-8">
+				<label for="sch-inp" class="wb-inv">Search [contextual]:</label>
+			</div>
+			<div class="col-md-4 text-right">
+				<ul class="list-inline">
+					<li><a href="/en/sr/st.html">Search Tips</a></li>
+					<li><a href="advanced-search-en.html">Advanced Search</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="input-group mrgn-tp-lg">
+			<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query aria-describedby="gc-pi">
+			<span class="input-group-btn">
+				<button type="submit" class="btn btn-primary"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search</span></button>
+			</span>
+		</div>
```

Comment:
For this search box, mor specifically for the submit button, please put the the magnifying glass icon without the word search to the right of the search box on the same line.